R/archive.R

```diff
@@ -665,21 +666,22 @@ epi_archive =
  
               # Note that we've already recycled comp value to make size stable,
               # so tibble() will just recycle time value appropriately
-              return(tibble::tibble(time_value = time_value, 
+              return(tibble::tibble(version = version, 
```

Comment:
Again, we've put such changes on hold / are still discussing; this should be `time_value = ref_time_value`.  (https://github.com/cmu-delphi/epiprocess/issues/163)